src/praisonai/praisonai/integrations/registry.py

```
@@ -0,0 +1,229 @@
+        
+        Returns:
+            Dict[str, bool]: Mapping of integration name to availability status
+        """
+        availability = {}
+        
+        for name, integration_class in self._integrations.items():
+            try:
+                # Create a temporary instance to check availability
+                instance = integration_class()
+                availability[name] = instance.is_available
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)